addon/ng2/blueprints/ng2/files/package.json

```diff
@@ -27,6 +31,7 @@
     "karma-chrome-launcher": "^0.2.1",
     "karma-jasmine": "^0.3.6",
     "protractor": "^3.0.0",
-    "typescript": "^1.7.3"
+    "typescript": "^1.7.3",
```

Comment:
maybe we should opt angular-cli users into the TS 1.8 beta. Then you get features like comments in the tsconfig.json which is really nice in a templated file - gives the user in-situ instructions for editing it